tests/php/modules/contact-form/test-class.grunion-contact-form.php

```diff
@@ -304,229 +353,307 @@ public function test_process_submission_will_store_fields_and_their_values_to_em
 	}
 
 	/**
+	 * Tests that the form subussion sends the correct single email.
+	 *
 	 * @author tonykova
 	 */
 	public function test_process_submission_sends_correct_single_email() {
-		// Fill field values
-		$this->add_field_values( array(
-			'name'     => 'John Doe',
-			'dropdown' => 'First option',
-			'radio'    => 'Second option',
-			'text'     => 'Texty text'
-		) );
+		// Fill field values.
+		$this->add_field_values(
+			array(
+				'name'     => 'John Doe',
+				'dropdown' => 'First option',
+				'radio'    => 'Second option',
+				'text'     => 'Texty text',
+			)
+		);
 
 		add_filter( 'wp_mail', array( $this, 'pre_test_process_submission_sends_correct_single_email' ) );
 
 		// Initialize a form with name, dropdown and radiobutton (first, second
-		// and third option), text field
-		$form = new Grunion_Contact_Form( array( 'to' => '"john" <john@example.com>', 'subject' => 'Hello there!' ), "[contact-field label='Name' type='name' required='1'/][contact-field label='Dropdown' type='select' options='First option,Second option,Third option'/][contact-field label='Radio' type='radio' options='First option,Second option,Third option'/][contact-field label='Text' type='text'/]" );
+		// and third option), text field.
+		$form   = new Grunion_Contact_Form(
+			array(
+				'to'      => '"john" <john@example.com>',
+				'subject' => 'Hello there!',
+			),
+			"[contact-field label='Name' type='name' required='1'/][contact-field label='Dropdown' type='select' options='First option,Second option,Third option'/][contact-field label='Radio' type='radio' options='First option,Second option,Third option'/][contact-field label='Text' type='text'/]"
+		);
 		$result = $form->process_submission();
 		$this->assertNotNull( $result );
 	}
 
-	public function pre_test_process_submission_sends_correct_single_email( $args ){
+	/**
+	 * This method is hooked to the wp-mail filter.
+	 *
+	 * @param array $args A compacted array of wp_mail() arguments, including the "to" email,
+	 *                    subject, message, headers, and attachments values.
+	 */
+	public function pre_test_process_submission_sends_correct_single_email( $args ) {
 		$this->assertContains( '"john" <john@example.com>', $args['to'] );
 		$this->assertEquals( 'Hello there!', $args['subject'] );
 
-		$expected = '<b>Name:</b> John Doe<br /><br />';
+		$expected  = '<b>Name:</b> John Doe<br /><br />';
 		$expected .= '<b>Dropdown:</b> First option<br /><br />';
 		$expected .= '<b>Radio:</b> Second option<br /><br />';
 		$expected .= '<b>Text:</b> Texty text<br /><br />';
 
-		// Divides email by the first empty line
+		// Divides email by the first empty line.
 		$email_body = explode( PHP_EOL . PHP_EOL, $args['message'] );
 		$email_body = $email_body[0];
 
 		$this->assertEquals( 0, strpos( $expected, $email_body ) );
```

Comment:
Here's a case that justifies the lint. The test has the args to `strpos` backwards and is looking at the wrong part of the email body, but since `0 == false` the test was still passing.
```suggestion
		$email_body = explode( PHP_EOL . PHP_EOL, $args['message'] );
		$email_body = $email_body[1];

		$this->assertStringStartsWith( $expected, $email_body );
```

The same thing happens above at line 352, but Github won't let me comment on that one.

Reply:
Thanks! I fixed up the remaining issues and removed the file from the exclude list in commit f94c0fa